### DISCUSSION TITLE
cmake MOD_AUTODETECT

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#2142] [General]
Is there some means in cmake build to disable automatic module detection?  I found this
```
if(DEFINED EXTRA_MODULES)
  list(APPEND MODULES ${EXTRA_MODULES})
endif()
```
cmake/modules.cmake but nothing about MOD_AUTODETECT.

**Comment by sreimers** [comment]
You can use this to override:
 ```
cmake -B build -DMODULES="ice;opus;g722;account;menu"
``` 

or this to extend:

```
cmake -B build -DEXTRA_MODULES="opus"
```
